lib/jnpr/jsnapy/check.py

```diff
@@ -603,7 +603,7 @@ def generate_test_files(
                         tfile)
                 if os.path.isfile(tfile):
                     test_file = open(tfile, 'r')
-                    tests_files.append(yaml.load(test_file))
+                    tests_files.append(yaml.load(test_file), Loader=yaml.FullLoader)
```

Comment:
@mtucker502 this needs to be updated `tests_files.append(yaml.load(test_file, Loader=yaml.FullLoader))`

Reply:
I should've waited until I was feeling better before working on this. Thanks for checking it for me.